lib/platform/github/gh-http-wrapper.ts

```diff
@@ -0,0 +1,144 @@
+import URL from 'url';
```

Comment:
I think this file should be `lib/util/http/github` and export a class `GithubHttp` which extends and overrides the `Http` class.